MaxText/train.py

```diff
@@ -273,8 +273,11 @@ def train_loop(config, state=None):
   local_metrics_file = open(config.metrics_file, 'a', encoding="utf8") if config.metrics_file else None
   running_gcs_metrics = [] if config.gcs_metrics else None
 
-  start_step = get_first_step(state)
+  start_step = max(get_first_step(state), config.profiling_start_step)
```

Comment:
Not sure I 'get it' -- isn't this changing our first step because we use `start_step` on 277?

I think you want this notion of start step on the check in 278? On this line we could instead write:

```
start_step = get_first_step(state) # this is the start_step for training
first_profiling_step = start_step + config.skip_first_n_steps_for_profiler
```

Reply:
Ah I see, yeah I will update. Thanks!